catroid/config/checkstyle.xml

```diff
@@ -172,31 +179,51 @@
     <module name="PackageDeclaration" />
     <module name="UnusedImports" />
 
+    <!-- Size -->
+    <module name="OuterTypeNumber" />
+
     <!-- Whitespace -->
     <module name="EmptyForInitializerPad" />
     <module name="EmptyForIteratorPad">
       <property name="option" value="space" />
     </module>
+    <module name="EmptyLineSeparator">
+      <property name="tokens" value="IMPORT, CLASS_DEF, INTERFACE_DEF"/>
+      <property name="allowNoEmptyLineBetweenFields" value="true"/>
+      <property name="allowMultipleEmptyLines" value="false"/>
+      <property name="allowMultipleEmptyLinesInsideClassMembers" value="false"/>
+    </module>
     <module name="GenericWhitespace" />
     <module name="MethodParamPad" />
     <module name="NoLineWrap" />
-    <module name="NoWhitespaceAfter">
-      <property name="tokens" value="BNOT, DEC, DOT, INC, LNOT, UNARY_MINUS, UNARY_PLUS, ARRAY_DECLARATOR" />
+    <module name="NoWhitespaceAfter" />
+    <module name="NoWhitespaceBefore">
+      <property name="allowLineBreaks" value="true"/>
+      <property name="tokens" value="COMMA, SEMI, POST_INC, POST_DEC, DOT, GENERIC_END, ELLIPSIS"/>
+    </module>
+    <module name="OperatorWrap">
+      <property name="option" value="nl"/>
+      <property name="tokens" value="QUESTION, COLON, EQUAL, NOT_EQUAL, DIV, PLUS, MINUS, STAR, MOD, SR, BSR, GE, GT, SL, LE, LT, BXOR, BOR, LOR, BAND, LAND, LITERAL_INSTANCEOF, TYPE_EXTENSION_AND, METHOD_REF"/>
+    </module>
+    <module name="OperatorWrap">
+      <property name="option" value="eol"/>
+      <property name="tokens" value="ASSIGN, DIV_ASSIGN, PLUS_ASSIGN, MINUS_ASSIGN, STAR_ASSIGN, MOD_ASSIGN, SR_ASSIGN, BSR_ASSIGN, SL_ASSIGN, BXOR_ASSIGN, BOR_ASSIGN, BAND_ASSIGN"/>
     </module>
-    <module name="NoWhitespaceBefore" />
-    <module name="OperatorWrap" />
     <module name="ParenPad" />
     <module name="SeparatorWrap">
-      <property name="tokens" value="DOT" />
+      <property name="tokens" value="DOT, AT" />
       <property name="option" value="nl" />
     </module>
     <module name="SeparatorWrap">
-      <property name="tokens" value="COMMA" />
+      <property name="tokens" value="COMMA, ELLIPSIS, ARRAY_DECLARATOR, RBRACK" />
       <property name="option" value="EOL" />
     </module>
+    <module name="SingleSpaceSeparator" />
```

Comment:
<img src="http://img.memecdn.com/i-made-this_o_901671.jpg" width="160">
…check and I'm really proud of it :smirk_cat: